### PR TITLE
Fix/mail api - Send group Invite with email

### PR DESCRIPTION
**Pull request by b4chh**
#### Description

This PR adds a template and a function for sending group invite emails. Specifically, I created a template called "SendGroupInviteMailContent" that can be used to define the content of the email. Then, I added a function called "SendGroupInviteMail" to the account service. This function is responsible for sending the email when a user invites another user to a group. The "SendGroupInviteMail" function is now connected to the notes service.

#### Changelog

- functions `SendGroupInviteMailContent` + `SendGroupInviteMail` ...

### CODE REVIEW

File: .env
Comment (gabriel654165) [inline .env:1]: Never push the .env !

**Review by gabriel654165 — COMMENTED** [review]
Excepted the .env everything fine

File: .env
Comment (b4chh) [reply .env:1]: Yes, this file has been deleted

**Comment by b4chh** [comment]
You said everything was good expected the .env file I removed the .env and remove also the InviteLink because this variable because this variable appeared when I merge fix/mail-api to main

**Pull request merged**